website/assets/images/permanent/README.md

```diff
@@ -16,12 +16,12 @@ To add an image, simply add an image to this folder using Fleet's standard image
 
 For example:
 ```
-images/permanent/icon-avatar-default-128x128@2x.png
+images/permanent/icon-avatar-default-128x128-2x.png
 ```
 
 Then, after merging, your image will be available from anywhere that can talk to the public internet!
 
 For example:
 ```
-<img alt="An unbreakable image of a lovely default avatar." src="https://fleetdm.com/images/permanent/icon-avatar-default-128x128@2x.png"/>
+<img alt="An unbreakable image of a lovely default avatar." src="https://fleetdm.com/images/permanent/icon-avatar-default-128x128-2x.png"/>
```

Comment:
@gillespi314 we normally use the `@`:
![image](https://user-images.githubusercontent.com/618009/131732387-0b595099-98ab-400e-9e58-5af9f6eaa1da.png)

Reply:
Yep, and that was how I had it before. But the Gravatar API redirect for some reason even though the @ was being properly URI encoded the Gravatar CDN redirect was stripping it out and breaking the whole default link. I spent awhile trying to make it work using the @ naming convention and I after investigation I believe the issue is on Gravatar's end so I figured it was probably a better use of time to just make an exception here so we can get this working and move on.

Reply:
ahh I see, gotcha.  Makes sense, thanks for explaining it for me (and posterity!)